klippy/gcode.py

```diff
@@ -156,19 +159,29 @@ def register_mux_command(self, cmd, key, value, func, desc=None):
         prev_values[value] = func
     def get_command_help(self):
         return dict(self.gcode_help)
+    def get_status(self, eventtime):
+        return {'commands': self.status_commands}
+    def _build_status_commands(self):
+        commands = {cmd: None for cmd in sorted(self.gcode_handlers)}
```

Comment:
This likely should be `commands = {cmd: {} for cmd in self.gcode_handlers}` as sorting isn't useful and it's preferable to use an empty dict instead of `None` for consistent typing.

Reply:
Done!

Reply:
FYI, still has the superfluous `sorted()`.

Reply:
Good point, removed now!